configure/cmake_win.cmake

```diff
@@ -165,8 +165,8 @@ install(DIRECTORY $ENV{OMNI_BASE}/include/omniVms DESTINATION include COMPONENT
 install(FILES $ENV{OMNI_BASE}/include/omniconfig.h DESTINATION include COMPONENT headers)
 install(FILES $ENV{OMNI_BASE}/include/omnithread.h DESTINATION include COMPONENT headers)
 install(FILES $ENV{ZMQ_BASE}/include/zmq.h DESTINATION include COMPONENT headers)
-install(FILES $ENV{ZMQ_BASE}/include/zmq.hpp DESTINATION include COMPONENT headers)
-install(FILES $ENV{ZMQ_BASE}/include/zmq_addon.hpp DESTINATION include COMPONENT headers)
+install(FILES $ENV{CPPZMQ_BASE}/include/zmq.hpp DESTINATION include COMPONENT headers)
+install(FILES $ENV{CPPZMQ_BASE}/include/zmq_addon.hpp DESTINATION include COMPONENT headers)
```

Comment:
There was this comment from @mliszcz at some point in this PR:

> 
> 
> I like the idea of separating zmq and cppzmq, but wouldn't this break other people's builds if they do not set CPPZMQ_BASE? Would it be possible to fall-back to using ZMQ_BASE if CPPZMQ_BASE is not defined? - just a suggestion, I have no experience in windows builds and do not know if this will be a problem.

What he was suggesting has been added in appveyor.yml but I think he also meant in the general case when compiling on Windows. 
What happens if CPPZMQ_BASE is not defined? If I understood well, I think he was suggesting to initialize CPPZMQ_BASE to ZMQ_BASE in this use case, but in the CMake file.

Reply:
Yes, I understood & answered that. My stance is still to let it break instead of of cluttering a pretty new, and mostly untested build system for backwards compatibility. TBH, with all the pain I had to go through to get it to compile, I would be pretty impressed with anyone else who build it on windows from scratch.